core/arxiv/submission/templates/submission-core/confirmation-email.html

```diff
@@ -0,0 +1,48 @@
+{% extends "mail/base.html" %}
+
+{% import "base/macros.html" as macros %}
+
+{% block email_title %}arXiv submission submit/{{ submission_id }}{% endblock email_title %}
+
+{% block message_title %}We have received your submission to arXiv.{% endblock message_title %}
+
+{% block message_body %}
+<p style="margin: 0; margin-bottom: 15px;">
+  Your temporary submission identifier is: <code>submit/{{ submission_id }}</code>.
+  You may update your submission at: <a href="{{ url_for("submission", submission_id=submission_id) }}">{{ url_for("submission", submission_id=submission_id) }}</a>.
+</p>
+
+<p style="margin: 0; margin-bottom: 15px;">
+  Your article is scheduled to be announced at {{ announce_time.strftime("%a, %-d %b %Y %H:%M:%S ET") }}. The abstract
+  will appear in the subsequent mailing as displayed below, except that the
+  submission identifier will be replaced by the official arXiv identifier.
+  Updates before {{ freeze_time.strftime("%a, %-d %b %Y %H:%M:%S ET") }} will not delay announcement.
+</p>
+
+<p style="margin: 0; margin-bottom: 15px;">
+  A paper password will be emailed to you when the article is announced. You
+  should share this with co-authors to allow them to claim ownership. If you
+  have a problem that you are not able to resolve through the web interface,
+  contact <a href="mailto:{{ config.SUPPORT_EMAIL }}">{{ config.SUPPORT_EMAIL }}</a> with a
+  description of the issue and reference the submission identifier.
+</p>
+
+{{ macros.abs(
+  arxiv_id,
+  submission.metadata.title,
+  submission.metadata.authors_display,
+  submission.metadata.abstract,
+  submission.created,
+  submission.primary_classification.category,
+  comments = submission.metadata.comments,
+  msc_class = submission.metadata.msc_class,
+  acm_class = submission.metadata.acm_class,
+  journal_ref = submission.metadata.journal_ref,
+  doi = submission.metadata.doi,
+  report_num = submission.metadata.report_num,
+  version = submission.version,
+  submission_history = [],
+  secondary_categories = submission.secondary_categories,
+  embed_stylesheet=1) }}
```

Comment:
as mentioned in submission-ui, it is safer to include this information as an unstyled section or possibly to write an inliner for styles. It's great info and more than we have now, worth including and possibly better to reduce delivery or display errors by doing plaintext data on this for all email. (with maybe a lightweight graphic element to set it off from the main body?)

Reply:
Also should label this as data that the user submitted. And test against TeX, or possibly might need to remove some of it for logistical simplicity.

Reply:
Ah crap, didn't think about TeX; maybe we can pre-render? Let's look at this